packages/eds-core-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -76,7 +77,7 @@ export type TooltipProps = {
   /** Tooltip placement relative to anchor */
   placement?: Placement
   /** Tooltip title */
-  title?: string
+  title?: string | ReactNode
```

Comment:
string is already included in ReactNode type so `string | ` can be removed here. Another thing is, `title` is also a valid html attribute (which is string), so for this to work, you need to replace` & HTMLAttributes<HTMLDivElement>` at the bottom of the type here with  `& Omit<HTMLAttributes<HTMLDivElement>, 'title'>`